README.rst

```diff
@@ -10,7 +10,7 @@ Development
 PySWMM Wiki
 	https://github.com/OpenWaterAnalytics/pyswmm/wiki/
 Cite our Paper
-	https://doi.org/10.5281/zenodo.3751574
+	McDonnell, Bryant E., Ratliff, Katherine M., Tryby, Michael E., Wu, Jennifer Jia Xin, & Mullapudi, Abhiram. (2020). PySWMM: The Python Interface to Stormwater Management Model (SWMM). *Journal of Open Source Software, 5*\ (52), 2292, https://doi.org/10.21105/joss.02292
```

Comment:
In addition to the nice clean version, I might suggest adding this as well:

```
@article{McDonnell2020,
  doi = {10.21105/joss.02292},
  url = {https://doi.org/10.21105/joss.02292},
  year = {2020},
  publisher = {The Open Journal},
  volume = {5},
  number = {52},
  pages = {2292},
  author = {Bryant E. McDonnell and Katherine Ratliff and Michael E. Tryby and Jennifer Jia Xin Wu and Abhiram Mullapudi},
  title = {PySWMM: The Python Interface to Stormwater Management Model (SWMM)},
  journal = {Journal of Open Source Software}
}
```

Reply:
@bemcdonnell do you know how we can make a link to copy that text rather than put all that text on the page? similar to how there is a link to copy the bibtex format on the right side of the [paper page](https://joss.theoj.org/papers/10.21105/joss.02292). I'm googling but haven't figured out how to do it yet

Reply:
Yeah I forgot how bad RST format was... Check out this.. this might work. If it doesn't on the first try, then just merge it without that haha

https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#code-block-directive

Reply:
@bemcdonnell I can't figure it out... I think we can add a link to the Bibtex format when the paper gets assimilated in Google Scholar. I'll go ahead and merge this now and add that to my to-do list when it's linked.